Ghidra/Features/Decompiler/src/decompile/cpp/slgh_compile.cc

```diff
@@ -2150,8 +2150,8 @@ string SleighCompile::checkSymbols(SymbolScope *scope)
   ostringstream msg;
   SymbolTree::const_iterator iter;
   for(iter=scope->begin();iter!=scope->end();++iter) {
+    if ((*iter)->getType() != SleighSymbol::label_symbol) continue;
```

Comment:
We don't know that it's a `LabelSymbol *` until after the `getType()` call. So the cast needs to come after.